packages/app/cypress/e2e/runner/reporter.errors.cy.ts

```diff
@@ -87,4 +86,908 @@ describe('errors ui', {
+      file,
+      column: 27,
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('exception', {
+      file,
+      column: 12,
+      message: 'bar is not a function',
+    })
+
+    verify.it('command failure', {
+      file,
+      column: 10,
+      message: 'Expected to find element: #does-not-exist, but never found it',
+    })
+  })
+
+  describe('cy.wrap', () => {
+    const file = 'wrap.cy.js'
+
+    before(() => {
+      setup({
+        fileName: file,
+        mockPreferredEditor: false,
+        onLoadStatsMessage: 'Failed:3',
+      })
+    })
+
+    verify.it('assertion failure', {
+      file,
+      column: 27,
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('exception', {
+      file,
+      column: 12,
+      message: 'bar is not a function',
+    })
+
+    verify.it('command failure', {
+      file,
+      column: 10,
+      message: 'Expected to find element: #does-not-exist, but never found it',
+    })
+  })
+
+  describe('cy.visit', () => {
+    const file = 'visit.cy.js'
+
+    before(() => {
+      setup({
+        fileName: file,
+        mockPreferredEditor: false,
+        onLoadStatsMessage: 'Failed:4',
+      })
+    })
+
+    verify.it('onBeforeLoad assertion failure', {
+      file,
+      column: 29,
+      codeFrameText: 'onBeforeLoad',
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('onBeforeLoad exception', {
+      file,
+      column: 14,
+      codeFrameText: 'onBeforeLoad',
+      message: 'bar is not a function',
+    })
+
+    verify.it('onLoad assertion failure', {
+      file,
+      column: 29,
+      codeFrameText: 'onLoad',
+      message: `expected 'actual' to equal 'expected'`,
+    })
+
+    verify.it('onLoad exception', {
+      file,
+      column: 14,
+      codeFrameText: 'onLoad',
+      message: 'bar is not a function',
+    })
+  })
+
+  describe('cy.intercept', () => {
+    const file = 'intercept.cy.ts'
+
+    before(() => {
+      setup({
+        fileName: file,
+        mockPreferredEditor: false,
+        onLoadStatsMessage: 'Failed:3',
+      })
+    })
+
+    verify.it('assertion failure in req callback', {
+      file,
+      column: 22,
+      message: [
+        `expected 'a' to equal 'b'`,
+      ],
+      notInMessage: [
+        'The following error originated from your spec code',
+      ],
+    })
+
+    verify.it('assertion failure in res callback', {
+      file,
+      column: 24,
+      codeFrameText: '.reply(()=>{',
+      message: [
+        `expected 'b' to equal 'c'`,
+      ],
+      notInMessage: [
+        'The following error originated from your spec code',
+      ],
+    })
+
+    verify.it('fails when erroneous response is received while awaiting response', {
+      file,
+      column: 6,
+      // this fails the active test because it's an asynchronous
+      // response failure from the network
+      codeFrameText: '81|.then(()=>{',
```

Comment:
is `81` the line number? Could this be impacted if a new test is added above it?

Reply:
81 is the line number and thus could be impacted. I don't expect these to change much but we can remove it and have a slightly looser assertion.